.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,11 @@
 Please fill in this template.
 
 - [ ] Use a meaningful title for the pull request. Include the name of the package modified.
+- [ ] Is this just a bug fix, new functionality, or breaking change?
 - [ ] Have read [How To Contribute](https://github.com/Turfjs/turf/blob/master/CONTRIBUTING.md#how-to-contribute).
 - [ ] Run `npm test` at the sub modules where changes have occurred.
+- [ ] Add tests for use cases addressed by the PR (if relevant)
+- [ ] Update JSDoc comments above changed functions (if relevant)
 - [ ] Run `npm run lint` to ensure code style at the turf module level.
```

Comment:
The test task a few lines above should have linted the code, so no need to include this as a seperate task.

Reply:
@smallsaucepan are you saying the linter updates the JSDoc comments above functions?  The intention here is to cover a couple more essential the contributor should provide for a complete PR.

Reply:
My understanding is the linter only regenerates the readmes from the JSDoc comments.  It's up to the developer to update the JSDoc comments themselves.

Reply:
I updated the template text a little bit to clarify what is expected.

Reply:
I feel like more should be in the PR checklist, but it also should be simple.  I just noticed the "how to contribute" link referenced in bullet 3 actually doesn't quite exist anymore.  I'm going to shape that up a little bit to pick up the slack of what should be in a good PR.  This PR has spidered a little bit, but I think in a good way.

Reply:
> @smallsaucepan are you saying the linter updates the JSDoc comments above functions?

More that running "npm test" runs "npm run lint" automatically anyway.

Reply:
Got it.  I've now moved these steps under a single linked doc  This makes the pull request template much simpler.

- [ ] Have read and followed the steps for [preparing a pull request](https://github.com/Turfjs/turf/blob/master/CONTRIBUTING.md#preparing-a-pull-request).